src/components/CustomerQuota/ItemsSelection/ItemIdentifier.tsx

```diff
@@ -78,13 +79,20 @@ export const ItemIdentifier: FunctionComponent<{
             disabled={scanButton.disabled}
             fullWidth={!textInput.visible}
             onPress={() => setShouldShowCamera(true)}
-            text={scanButton.text || "Scan"}
+            text={
+              scanButton.text ||
+              i18n.t("customerQuotaScreen.quotaIdentifierButtonScan")
+            }
           />
         )}
       </View>
       {shouldShowCamera && (
         <IdentifierScanModal
-          cancelButtonText={textInput.disabled ? "Back" : "Enter manually"}
+          cancelButtonText={
+            textInput.disabled
+              ? i18n.t("customerQuotaScreen.quotaScanButtonBack")
+              : i18n.t("customerQuotaScreen.enterManually")
+          }
```

Comment:
When I tried on stg to scan for First voucher code, I got the screen below. But didn't get it when trying to scan for TT token (oh because TT token is "Back" and not "Enter manually")

![image](https://user-images.githubusercontent.com/12404066/94260086-1d334a80-ff62-11ea-900a-f4ff7a029e77.png)

Reply:
Whoops this was supposed to be "idScanner.enterManually"!